ci/lepton/model_convergence/scripts/wrap_template.sh

```
@@ -0,0 +1,271 @@
+keys = [s.strip() for s in os.environ.get("GPU_FIELDS","").split(",") if s.strip()]
+rows = []
+for r in csv.reader(sys.stdin):
+    if not r:
+        continue
+    vals = [x.strip() for x in r]
+    if len(vals) < len(keys):
+        vals += [None]*(len(keys)-len(vals))
+    rows.append(dict(zip(keys, vals[:len(keys)])))
+print(json.dumps(rows))
+PY
+    <<< "$RAW_SMI"
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)